katas/content/random_numbers/weighted_random_bit/verification.qs

```diff
@@ -46,10 +46,10 @@ namespace Kata.Verification {
             }
         } else {
             if zeroCount < goalZeroCount - 4 * nRuns / 100 {
-                Message($"Unexpectedly low number of 0's generated: expected around {x * IntAsDouble(nRuns)} 0's, got {zeroCount} out of {nRuns}");
+                Message($"Unexpectedly low number of 0's generated : expected around {x * IntAsDouble(nRuns)} 0's, got {zeroCount} out of {nRuns}");
```

Comment:
We even do this in messages?

Reply:
I tried to avoid changing messages and comments. Looks like I missed these two.

Reply:
should be fixed now